morphman/manipulate_area.py

```diff
@@ -274,15 +274,15 @@ def change_area(voronoi, factor, line_to_change, diverging_centerline, diverging
         # Get direction (v_change) to move the point
         AC_length = np.linalg.norm(AC)
         if AC_length != 0:
-            h = np.linalg.norm(np.cross(BA, BC)) / AC_length # shortest distance between point and line
-            D = A + sign * (AC / AC_length) * np.sqrt(np.linalg.norm(BA)**2 - h**2)
+            h = np.linalg.norm(np.cross(BA, BC)) / AC_length  # shortest distance between point and line
+            D = A + sign * (AC / AC_length) * np.sqrt(np.linalg.norm(BA) ** 2 - h ** 2)
         else:
             D = A
 
         v_change = D - B
         if sign > 0:
-            factor_ = factor[tmp_id1] * (1 - np.linalg.norm(A-D) / AC_length) \
-                    + factor[tmp_id2] * (np.linalg.norm(A-D) / AC_length)
+            factor_ = factor[tmp_id1] * (1 - np.linalg.norm(A - D) / AC_length) \
+                      + factor[tmp_id2] * (np.linalg.norm(A - D) / AC_length)
             if factor_ > 10:
                 print(factor_, factor[tmp_id1], factor[tmp_id2], AC_length,
                       np.linalg.norm(c_change))
```

Comment:
Kan ikke se at `c_change` er definert. Skulle det vært:
```suggestion
                      np.linalg.norm(v_change))
```

Reply:
Det stemmer nok, men mer generelt så burde hele den if-testen tas bort. Jeg antar at du har gjort det i asymmetric branchen?

Reply:
Ja, skulle til å spørre hva denne snutten gjorde. Det divergerer heftig godt med bulge / stenosis, så gjør en sjekk på metode før if testen i asymmetric-branchen. Men kan godt fjerne hele greia.